bundles/emfprofiles.custom/src/emfprofiles/custom/wizards/EMFProfilesProjectOperation.java

```diff
@@ -99,167 +90,140 @@ protected void execute(IProgressMonitor monitor) throws CoreException,
 //        subMonitor.subTask("Opening representation");
 //        openRepresentation();
```

Comment:
you should remove these comments as well, as the openRepresentation() is not there anymore.

Reply:
Right but it is always a trade-off between introducing an additional dependency and reusing code.